packages/manager/src/features/IAM/Shared/NoAssignedRoles/NoAssignedRoles.tsx

```diff
@@ -0,0 +1,31 @@
+import { Paper, Typography } from '@linode/ui';
+import Grid from '@mui/material/Grid';
+import React from 'react';
+
+import EmptyNotificationIcon from 'src/assets/icons/emptynotification.svg';
+
+interface Props {
+  text: string;
+}
+
+export const NoAssignedRoles = (props: Props) => {
+  const { text } = props;
+
+  return (
+    <Paper variant="outlined">
+      <Grid
+        sx={{
+          alignItems: 'center',
+          justifyContent: 'center',
+          paddingBottom: 6,
+          paddingTop: 6,
+        }}
+        container
+        direction="column"
+      >
+        <EmptyNotificationIcon />
+        <Typography sx={{ marginTop: 1 }}>{text}</Typography>
+      </Grid>
+    </Paper>
```

Comment:
Is the use of `Grid` intentional here? Usually we like to reserve `Grid` for complex layouts only. In this case, you could just apply the intended styles to the `Paper`

```suggestion
    <Paper
      sx={{
        alignItems: 'center',
        display: 'flex',
        flexDirection: 'column',
        justifyContent: 'center',
        paddingY: 16,
      }}
      variant="outlined"
    >
      <EmptyNotificationIcon />
      <Typography sx={{ marginTop: 1 }}>{text}</Typography>
    </Paper>
```

Or alternatively, you could use a `Stack`

```suggestion
    <Paper variant="outlined">
      <Stack alignItems="center" paddingY={10} spacing={2}>
        <EmptyNotificationIcon />
        <Typography sx={{ marginTop: 1 }}>{text}</Typography>
      </Stack>
    </Paper>
```

Reply:
sure, done